openpype/hosts/houdini/api/lib.py

```diff
@@ -460,3 +476,60 @@ def reset_framerange():
     hou.playbar.setFrameRange(frame_start, frame_end)
     hou.playbar.setPlaybackRange(frame_start, frame_end)
     hou.setFrame(frame_start)
+
+
+def load_creator_code_to_asset(
+        otl_file_path, node_type_name, source_file_path):
```

Comment:
Can we please add a docstring to describe what this function does?

Reply:
this was from the early attempts. Idea was to have creators as nodes and use Houdini to execute creator python code on them. It was unnecessary complicated and ... useless :) so this is now removed.